app-frontend/src/app/components/common/search/search.js

```diff
@@ -26,6 +27,9 @@ class SearchController {
     }
 
     $postLink() {
+        if (this.$stateParams.search) {
```

Comment:
This seems a bit unintuitive to me and sort of breaks the whole "components shouldn't fetch their own data" rule.

Reply:
that's a good point -and not actually necessary since I added a `value` binding anyways